clang/lib/AST/Interp/Interp.h

```diff
@@ -209,7 +209,7 @@ bool Ret(InterpState &S, CodePtr &PC, APValue &Result) {
     // FIXME: We could be calling isLive() here, but the emitted diagnostics
     // seem a little weird, at least if the returned expression is of
     // pointer type.
```

Comment:
Can we add to the comment that we are special casing `nullptr` b/c they are not considered alive?